frontend/ui/widget/confirmbox.lua

```diff
@@ -154,7 +154,6 @@ function ConfirmBox:init()
 
     local frame = FrameContainer:new{
         background = Blitbuffer.COLOR_WHITE,
-        margin = self.margin,
```

Comment:
Elsewhere where self.margin is used isn't messed up by this singular removal?

Reply:
May be, it is added to the width of a VerticalSpan.
Or may be not: there, it is used as some vertical "width", for separation between the text and the bottom buttontable. `Size.margin.default + Size.padding.default`. A bit ugly.
~~I think it's best to either set `margin = 0` instead of `Size.margin.default` in InputContainer:extend{} (this would allow other widget to set it), or remove every occurence of `margin` / `self.margin`.~~
(InfoMessage does not have any such `margin`)